platform/cli/src/index.js

```diff
@@ -32,17 +33,9 @@ function getOptionsFromAnswers(answers) {
 
 program.version('0.0.1').description('OHIF CLI');
 
-// For debugging
-program
-  .command('debug <name> <license>')
-  .description('Create a new extension')
-  .action((name, license) => {
-    createExtension({ name, license });
-  });
-
 program
   .command('create-extension')
-  .description('Create a new extension')
+  .description('Create a new template extension')
```

Comment:
Hmm, I kinda liked "Create a new extension", its already somewhat customized by your input.

If you want to include the word template, maybe: "Create new extension from template"?